config.go

```diff
@@ -249,6 +250,8 @@ func xmlToFileLogWriter(filename string, props []xmlProperty, enabled bool) (*Fi
 			daily = strings.Trim(prop.Value, " \r\n") != "false"
 		case "rotate":
 			rotate = strings.Trim(prop.Value, " \r\n") != "false"
+		case "integersuffix":
+			integerSuffix = strings.Trim(prop.Value, " \r\n") != "false"
```

Comment:
As long as the user sees the same behavior then we are good.  How are we planning on manipulating the default behavior from within the agent?